general/standards/electrical-standard.rst

```diff
@@ -0,0 +1,4 @@
+Electrical Standard
+===================
+
+Refer to this `Google Doc <https://docs.google.com/document/d/1Dor3P09H9-mdcJvkMIZcK-YoCY1UyPBDU1qtubGDoAM/edit>`_
```

Comment:
Can we pleeeeeeeease copy the contents of the doc onto the site? For me?

Reply:
Will fix in future PR